hack/test-cmd.sh

```diff
@@ -34,6 +34,30 @@ function find_tests() {
 }
 tests=( $(find_tests ${1:-.*}) )
 
+# install_image_pull_secret will look for ~/docker/config.json and create an image pull secret.  If no suitable credentials are found a warning will be displayed.  It assumes a running server.
+#
+# Globals:
+#  - ADMIN_KUBECONFIG
+#  - REAL_HOME
+# Arguments:
+#  None
```

Comment:
@vrutkovs doesn't it take an optional namespace as an argument?

Reply:
Good catch, thanks. Fixed this in 6757e9f